modules/proto/test/src/smithytranslate/proto3/internals/CompilerRendererSuite.scala

```diff
@@ -1621,17 +1668,16 @@ class CompilerRendererSuite extends FunSuite {
                       |
                       |package test;
                       |
-                      |import "google/protobuf/timestamp.proto";
-                      |
                       |import "alloy/protobuf/wrappers.proto";
                       |
                       |import "alloy/protobuf/types.proto";
                       |
                       |message MyStructure {
-                      |  google.protobuf.Timestamp basic = 1;
+                      |  string basic = 1;
                       |  alloy.protobuf.OffsetDateTimeValue wrapped = 2;
-                      |  alloy.protobuf.CompactOffsetDateTime compact = 3;
-                      |  alloy.protobuf.CompactOffsetDateTimeValue wrappedCompact = 4;
+                      |  string defaultWithNoFormat = 3;
```

Comment:
I'm not exactly clear on why this `defaultWithNoFormat` is a String. I thought based on this comment: https://github.com/disneystreaming/alloy/blob/4824e9d4cd467e8a660b247f72a7ef37303f7c0a/modules/core/resources/META-INF/smithy/proto/proto.smithy#L61 that it would be `message { int64 seconds = 1; int32 nanos = 2; string zone_offset = 3; }`.

Perhaps that comment is wrong though or I am misinterpreting it